scapy/contrib/isotp/isotp_native_socket.py

```diff
@@ -329,7 +333,11 @@ def __init__(self,
                                        stmin=stmin, bs=bs))
         self.can_socket.setsockopt(SOL_CAN_ISOTP,
                                    CAN_ISOTP_LL_OPTS,
-                                   self.__build_can_isotp_ll_options())
+                                   self.__build_can_isotp_ll_options(
```

Comment:
I think, providing a new funtion ```__build_canfd_isotp_ll_options``` would be more clean

Reply:
That would be just a copy of the __build_can_isotp_ll_options function which also has CANFD option mentioned in its comments. Please think again about this :)